packages/ripple-binary-codec/HISTORY.md

```diff
@@ -2,8 +2,11 @@
 
 ## Unreleased
 
+## 2.3.0 (2025-2-13)
+
 ### Added
 * Support for the AMMClawback amendment (XLS-73)
+* Support for the Permissioned Domains amendment (XLS-80).
```

Comment:
`PermissionedDomain` amendment does not affect `ripple-binary-codec` anymore than every other amendment. Is it required to include this line in the CHANGELOG ?

Reply:
Yes it's required because `definitions.json` change is within `ripple-binary-codec`